src/main/java/mServer/tool/DateWithoutTimeComparer.java

```diff
@@ -0,0 +1,16 @@
+package mServer.tool;
+
+import java.util.Calendar;
+
+public class DateWithoutTimeComparer {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)